ingestr/src/appsflyer/_init_.py

```diff
@@ -11,14 +13,38 @@
 def appsflyer_source(
     api_key: str, start_date: str, end_date: str
 ) -> Iterable[DltResource]:
+    start_date_obj = ensure_pendulum_datetime(start_date)
     client = AppsflyerClient(api_key)
 
-    @dlt.resource(write_disposition="merge", merge_key="install_time")
-    def campaigns() -> Iterable[TDataItem]:
-        yield from client.fetch_campaigns(start_date, end_date)
+    @dlt.resource(write_disposition="merge", merge_key="Install Time")
+    def campaigns(
+        updated=dlt.sources.incremental('["Install Time"]', start_date_obj.isoformat()),
+    ) -> Iterable[TDataItem]:
+        current_start_time = datetime.fromisoformat(updated.start_value).date()
```

Comment:
shouldn't this be `updated.last_value`?

Reply:
While testing, both start_value and last_value were giving the same value, so I used start_value.